library/src/scala/IArray.scala

```diff
@@ -363,16 +417,32 @@ object IArray:
   /** An immutable array with given elements. */
   def apply(x: Unit, xs: Unit*): IArray[Unit] = Array(x, xs: _*)
 
+  /** Build an array from the iterable collection.
+   *
+   *  {{{
+   *  scala> val a = Array.from(Seq(1, 5))
+   *  val a: Array[Int] = Array(1, 5)
+   *
+   *  scala> val b = Array.from(Range(1, 5))
+   *  val b: Array[Int] = Array(1, 2, 3, 4)
+   *  }}}
+   *
+   *  @param  it the iterable collection
+   *  @return    an array consisting of elements of the iterable collection
+   */
+  def from[A : ClassTag](it: IterableOnce[A]): Array[A] =
+    Array.from(it)
+
+  def newBuilder[T](using t: ClassTag[T]): IArrayBuilder[T] =
+    IArrayBuilder.make[T]
+
   /** Concatenates all arrays into a single immutable array.
    *
    *  @param xss the given immutable arrays
    *  @return   the array created from concatenating `xss`
    */
   def concat[T: ClassTag](xss: IArray[T]*): IArray[T] =
-    // `Array.concat` should arguably take in a `Seq[Array[_ <: T]]`,
-    // but since it currently takes a `Seq[Array[T]]` we have to perform a cast,
-    // knowing tacitly that `concat` is not going to do the wrong thing.
-    Array.concat[T](xss.asInstanceOf[Seq[Array[T]]]: _*)
+    Array.concat[T](xss.asInstanceOf[immutable.Seq[Array[T]]]: _*)
```

Comment:
Same here: the comment should probably stay. Also, why the change `Seq` -> `immutable.Seq` (they're the same)?

Reply:
I added the import to `import scala.collection._` which shadowed the default one.

Reply:
That is usually a dangerous import, which is typically not recommended. It means that you can get the non-immutable types by default in all sorts of places without meaning too. Consider either important exactly what you need, like
```scala
import scala.collection.Iterable
```
or just use `collection.Iterable` when you need the non-immutable ones.

Reply:
Will do